is-array/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for is-array 1.0
+// Project: https://github.com/retrofox/is-array
+// Definitions by: Pine Mizune <https://github.com/pine>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = isArray;
+declare function isArray(val?: any): boolean;
```

Comment:
`isArray` can also receive not array types. And it returns boolean always.

https://github.com/retrofox/is-array/blob/master/test/test.html
https://github.com/retrofox/is-array/blob/master/index.js#L14-L29